randovania/games/dread/json_data/Artaria.txt

```diff
@@ -774,6 +778,8 @@ Extra - asset_id: collision_camera_008
   * Extra - right_shield_def: None
   > Door to White EMMI Introduction (Thermal)
       Trivial
+  > Teleporter to Dairon - Teleport to Artaria
+      Wave Beam and Wide Beam
```

Comment:
Just needs Wave Beam if this gets accepted. Wave has properties of Wide

Reply:
There could be a time where beam properties get separated, in which case this could be bugged logic expecting you to shoot the door with wide beam that you don't have.

Reply:
There is, but we dont need to worry about that right now

Reply:
And there's a Misc for Disabled Beam Separation, but i asked someone about accounting for that and they said not to worry about it